### ISSUE TITLE
SIP does not work

### ISSUE THREAD

**Issue by flaviogoncalves** [livekit/livekit#2937]
**Describe the bug**
I tried everything to make the SIP inbound work. I got to call and the system never answers. From webrtc it is working. If I change the codec to Opus I receive the error 400.

**Server**
- Version: Cloud

**Client**
SIP, FreeSwitch

**To Reproduce**
Create a trunk to the livekit endpoint and send a call. Either it rings and does not answer, either generates 400 Bad Request error.  I can see the agent being activated but no answer in the call. 

**Expected behavior**
Answer call and transfer to the agent

**Screenshots**
![image](https://github.com/user-attachments/assets/7fb4fc75-0021-4047-b3b0-473a7e69c10e)

![image](https://github.com/user-attachments/assets/070beeae-995b-4ba0-a360-a9c315701ce0)

**Additional context**

**Comment by raihan0824** [comment]
hi, did you solver this? i also have the same problem

**Comment by flaviogoncalves** [comment]
I have used G711 ulaw ang G722 I did not tried again with Opus. Also some of these 400 errors are not actually a Bad Request some of them are 404 or even 403. I'm not sure if this changed, but all the errors where generic. So check the number dialed, ip address and source number. The last version I've checked there was no differentiation between 488 codec mismatch, 404 not found or 403 Forbidden, everything returned 400.

**Comment by yepher** [comment]
I thought +55 (Brazil) used A-Law companding for G711.

**Comment by flaviogoncalves** [comment]
Yes, you are right Brasil uses A-law in the PSTN, but I was testing using a simple PBX. I haven't tested with A-law. It would be a nice try.

**Comment by HitheshShty** [comment]
Adding the SipTrunkID in the dispatch_rule.json might sole the problem
